### ISSUE TITLE
Creating Stage on EaselJS 1.0.2 doesn't work on iOS 8.3 / Safari (0.8.2 does work)

### ISSUE THREAD

**Issue by Friksel** [CreateJS/EaselJS#982]
I recently found out some of my animations using EaselJs were and others weren't playing on iOS 8.3. iPad Air in Safari (but probably other browsers on iOS 8.3 neither). I was surprised to see the animations not even showing are build on EaselJs v1.0.2 and the animations that are fully working without problems are build on 0.8.2.

1.0.2 halts on creating the stage. According to the API creating the Stage is done the same way in 1.0.2 as in 0.8.2 and that makes sense, 'cause in all other devices I tested this on (and different browsers on those devices) just work fine with 1.0.2.

I had to move back to version 0.8.2 to support iOS 8.3 again, and now they work fine there too.
So something seems to be off in EaselJS 1.0.2.
I'm surprised to see this version is a year old and can't find any complaints about iOS problems on the issues here. iOS doesn't seem THAT old, 4 years. Or is the support dropped for iOS 8 from version 1?

**Comment by joseph-olivemultimedia** [comment]
Yes, i do get the same issue, I was not able to view the files in iOS

**Comment by lannymcnie** [comment]
Thanks, will check it out.

**Comment by Friksel** [comment]
This issue was opened in september, so about 3 months ago, and is about iOS not working at all with EaselJs, so I'm a little bit surprised there's no fix for this yet. Projects here still have to go with a very old EaselJs version because of this. Is there any progress on this? Thanks
